.github/workflows/processor-release.yml

```diff
@@ -0,0 +1,67 @@
+name: processor-release
+
+on:
+  release:
+    types: [ created ]
+
+jobs:
+  release:
+    name: processor release - ${{ matrix.target }}
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            runner: ubuntu-24.04
+            os: ubuntu
+            cross: false
+            python-version: 3.13
```

Comment:
I double checked the last artifact and it looks like it is linked to 3.12 for the Linux x86-64 build.

Reply:
```$ ldd rotel
	linux-vdso.so.1 (0x00007f1c6a3c0000)
	libpython3.12.so.1.0 => not found
	libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f1c6a374000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f1c6a28e000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f1c6940d000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f1c6a3c2000)
```

Reply:
oh, got confused by your message in slack about `looks good except my box is on 3.13`. We can do another matrix entry with 24.04 but use the setup- action to install python 3.13, sound good?

Reply:
Yeah, once we have the setup- action in place it seems like we can target any version even if it isn't the base one.